Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,261 @@
+
+In order to receive updates about GSoC at Oppia, please subscribe to the [Oppia GSoC Announce](https://groups.google.com/g/oppia-gsoc-announce) mailing list.
+
+Oppia will strictly adhere to GSoC timeline and will not be able to provide any additional extensions. Please refer to 
+the [Dates and Deadlines](#dates-and-deadlines) section below.
+
+Also, please note that acceptance into GSoC isn't a prerequisite for becoming an Oppia contributor. The Oppia project is run by the community for the community, and we warmly welcome anyone who'd like to help out! You can get started by following the instructions [here](https://github.com/oppia/oppia/wiki).
+
+
+# Contributors
+
+GSoC is an excellent opportunity for new contributors to get paid to work on an open source project. If you're interested in applying as a contributor, you should definitely read the following resources:
+
+-   [Google Summer of Code contributor guide](https://google.github.io/gsocguides/student/)
+-   [Google's list of resources](https://developers.google.com/open-source/gsoc/resources/)
+-   [GSoC FAQ](https://developers.google.com/open-source/gsoc/faq)
+
+
+## Getting started
+
+Welcome! If you're interested in applying to work with Oppia for GSoC, please follow these steps:
+
+1. Sign up to the [oppia-gsoc-announce@](https://groups.google.com/forum/#!forum/oppia-gsoc-announce) mailing list in order to receive important notifications about Oppia's participation in GSoC. If you like, you can turn on the notification for github discussions to  participate in general discussion related to Oppia's involvement in GSoC. Make sure to set your preferences correctly so that you actually get the emails!
+
+2. Get a better understanding of what Oppia is about:
+    - Read the [user documentation](http://oppia.github.io/#/) to become familiar with important concepts like explorations and interactions.
+    - Play some lessons on [Oppia.org](https://www.oppia.org/learn/math), which hosts a live instance of Oppia.
+
+3. To get started with development, read and follow the instructions in the contributors' guide carefully ([Oppia Web](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up)).
```

Comment:
This seems like a good place to call out Oppia Android's non-participation this year. 

Maybe change this last part to: "...in the Oppia Web contributors' guide carefully. (Note that Oppia Android won't be offering GSoC projects in 2023, but will be doing so again from 2024 onwards.)" with the link put on "Oppia Web contributors' guide".

Reply:
Added